README.md

```diff
@@ -37,6 +37,57 @@ cmake .. -DENABLE_Z3=1 -DC2GOTO_SYSROOT=/Library/Developer/CommandLineTools/SDKs
 make -j8
 ````
 
+However, ESBMC is not as stable as the Linux version. Hence, we recommend using AMD64 via docker for optimal performance, until full compatibility is achieved. Sample docker compose and dockerfiles follow:
```

Comment:
What does it mean that ESMBC is not as stable as the Linux version? Are there any open bugs for those stability issues?

Reply:
Good question. I have many issues I can report, but I did not do so yet, because for example even Float16 does not work to my knowledge